packetbeat/_meta/fields.yml

```diff
@@ -79,10 +79,18 @@
       description: >
         The name of the process that served the transaction.
 
+    - name: cmdline
```

Comment:
So these are not new, correct? They were just missing from fields.yml before?

Reply:
Do we have something like the `fields_check` we have for metricbeat in packetbeat to detect these?

Reply:
Yes, those were missing

Reply:
@ruflin The packetbeat system tests all appear to be checking that the fields are documented. But there are no tests with live data for the process enrichment feature (the feature is disabled when reading from pcaps since that isn't live).